README.rst

```diff
@@ -20,14 +20,14 @@ This client supports the following Google Cloud Platform services:
 -  `Google Cloud Pub/Sub`_
 -  `Google BigQuery`_
 -  `Google Cloud Resource Manager`_
--  `Google Cloud Logging`_
+-  `Google Stackdriver Logging`_
```

Comment:
I don't *think* it matters but the branding guidelines say long name is "Stackdriver Logging". But the long name for Stackdriver says "Google Stackdriver." So unclear if this should be "Google Stackdriver Logging" or "Stackdriver Logging" @sharbison3 ?